packages/webapp/pages/onboarding.tsx

```diff
@@ -233,18 +259,85 @@ export function OnboardPage(): ReactElement {
     <Container className="flex-col">
       <NextSeo {...seo} titleTemplate="%s | daily.dev" />
       <ProgressBar percentage={isFiltering ? percentage : 0} />
-      <Logo
-        className="py-8 px-10 w-auto laptop:w-full"
-        position={LogoPosition.Relative}
-      />
+      <header
+        className={classNames(
+          'flex justify-between w-full h-full flew-row px-6',
+          !isAuthenticating && !isFiltering && 'mt-20 w-full max-w-[75rem]',
+        )}
+      >
+        <Logo
+          className={
+            !isAuthenticating && !isFiltering
+              ? 'w-auto'
+              : 'py-8 px-10 w-auto laptop:w-full'
+          }
+          logoClassName={
+            !isAuthenticating && !isFiltering ? 'h-logo-big' : 'h-logo'
+          }
+          position={LogoPosition.Relative}
+        />
+
+        {!isAuthenticating && !isFiltering && (
+          <span
+            className={classNames(
+              'flex items-center',
+              'text-theme-label-tertiary',
+            )}
+          >
+            <span className="hidden tablet:block">
+              Already a daily.dev member?
+            </span>
+            <Button
+              className="ml-3 btn-secondary"
+              onClick={() =>
+                setAuth({ isAuthenticating: true, isLoginFlow: true })
+              }
+            >
+              Log in
+            </Button>
+          </span>
+        )}
+      </header>
       <div
         className={classNames(
-          'flex relative flex-col flex-1 items-center w-full max-h-[40rem]',
-          isFiltering
-            ? 'laptop:max-w-[48.75rem] tablet:max-w-[32rem]'
-            : containerClass,
+          'flex flex-wrap justify-center px-6 mt-8 laptop:mt-20 w-full max-w-[75rem]',
+          !isAuthenticating && 'flex-1',
         )}
       >
+        {!isFiltering && !isAuthenticating && (
+          <div
+            className={classNames(
+              'flex flex-1 flex-col laptop:max-w-[27.5rem] laptop:mr-8',
+            )}
+          >
+            <OnboardingTitle className="typo-large-title tablet:typo-mega1">
+              Where developers grow together
+            </OnboardingTitle>
+
+            <h2 className="mb-8 typo-body tablet:typo-title2">
+              Get one personalized feed for all the knowledge you need.
+            </h2>
+
+            {getAuthOptions(AuthDisplay.OnboardingSignup)}
+
+            <div className="flex relative flex-col tablet:flex-row flex-1 justify-end tablet:items-center pb-6 tablet:pb-0 mt-8 tablet:mt-auto w-full tablet:max-h-[10rem]">
+              <SignupDisclaimer className="mb-auto tablet:mb-0" />
```

Comment:
In this section the signup disclaimer is not clickable.

Reply:
Fixed it, just z-index on the image